src/dbt_score/dbt_utils.py

```diff
@@ -31,8 +42,13 @@ def dbt_parse() -> dbtRunnerResult:
     Raises:
         DbtParseException: dbt parse failed.
     """
+    if not DBT_INSTALLED:
+        raise DbtNotInstalledException(
+            "To execute dbt parse, install dbt-core in your environment."
```

Comment:
```suggestion
            "This option requires dbt to be installed in the same Python environment as dbt-score."
```
Maybe this phrasing?

Reply:
I like it!